examples/src/Examples.js

```diff
@@ -83,10 +85,23 @@ export default class Examples extends Component {
     return null;
   }
 
+  processEvents() {
+    if (eventQueue.length > 0) {
+      let event = eventQueue[0];
```

Comment:
Could do `let event = eventQueue.shift()` here, and in the callback below get rid of `eventQueue.shift();`.